packages/cf-style-container/src/index.js

```diff
@@ -37,4 +39,24 @@ const applyTheme = (ComponentToWrap, theme) => {
   return ThemedComponent;
 };
 
-export { createComponent, applyTheme, ThemeProvider, color };
+const createComponentStyles = (styleFunctions, component) => {
+  const mapStylesToProps = props =>
+    renderer => {
+      const toRender = {};
+      for (const style in styleFunctions) {
+        toRender[style] = renderer.renderRule(styleFunctions[style], props);
+      }
+      return toRender;
+    };
+  return connect(mapStylesToProps)(component);
+};
+
+export {
+  createComponent,
+  applyTheme,
+  ThemeProvider,
+  color,
+  connect,
+  combineRules,
+  createComponentStyles
+};
```

Comment:
I still don't know why this package is called cf-style-container? It doesn't contain any styles. This looks like cf-util-styles to me.